include/openssl/core_names.h

```diff
@@ -185,6 +185,10 @@ extern "C" {
 #define OSSL_PKEY_PARAM_DSA_PUB_KEY  "pub"
 #define OSSL_PKEY_PARAM_DSA_PRIV_KEY "priv"
 
+/* ECX Keys */
+#define OSSL_PKEY_PARAM_ECX_PUB_KEY  "pub"
```

Comment:
Maybe there should be a common define that is shared for these..

Reply:
Done